src/Rec/Rec.ts

```diff
@@ -529,3 +529,20 @@ export const ofRecord = <K extends string, V extends NonNullish>(
         (rec, [k, v]) => pipe(rec, set([k as K, v], equalityComparer)),
         empty()
     )
+
+/**
+ * Merges the key/value pairs from `Rec` b into `Rec` a. If a key already exists
+ * in `Rec` a, the merged `Rec` will include the value from `Rec` b.
+ *
+ * @group Transformations
+ *
+ * @returns A new `Rec` with the merged values
+ */
+// TODO: should it be merge or mergeInto?
+// TODO: does this need an equality comparer?
```

Comment:
@taylorn16 If you can please answer these two questions, I will try to get this completed.

Reply:
Sorry lost track of this!

> should it be merge or mergeInto?
I think `mergeInto` because order matters, and this gives a better sense of what is going to be merged into what in a function `pipe`line. `mergeInto` reads like, I'm going to merge the previous result in the pipeline into this record, whereas `merge` is ambiguous about which one is first and second in the spread.

> does this need an equality comparer?
No, because the keys are `K extends string`. Because these are just plain JS objects at runtime, we can't do any custom equality comparison on keys like we can with `Map`s.